javascript/build.xml

```diff
@@ -135,6 +135,7 @@
     <gjslint inputfile="i18n/phonenumbers/asyoutypeformatter*.js" />
```

Comment:
BTW: It looks like shortnumberinfo_test.js is not in any test target yet.
I think we need to add it to <target name="compile-tests" ... >

Reply:
Done. I also cleaned up the test to remove unused requires and make the compilation succeed.